interface/themes/oe-styles/style_light.scss

```diff
@@ -5,8 +5,6 @@ $close-color: $black;
 $close-text-shadow: 0 1 0 $black;
 $gray: #cdcdcd;
 
-@import "../default-variables";
```

Comment:
This is what broke light theme. I knew there was a reason this was patched out in past. Now we know why.